apps/web/modules/apps/[slug]/[...pages]/pages-view.tsx

```diff
@@ -55,48 +48,21 @@ function getRoute(appName: string, pages: string[]) {
 }
 export type PageProps = inferSSRProps<typeof getServerSideProps>;
 
-const AppPage: AppPageType["default"] = function AppPage(props: PageProps) {
+function AppPage(props: PageProps) {
   const appName = props.appName;
   const params = useParamsWithFallback();
   const pages = Array.isArray(params.pages) ? params.pages : params.pages?.split("/") ?? [];
   const route = getRoute(appName, pages);
 
+  if (!route || route.notFound) {
+    throw new Error("Route can't be undefined");
+  }
+
   const componentProps = {
     ...props,
     pages: pages.slice(1),
   };
-
-  if (!route || route.notFound) {
-    throw new Error("Route can't be undefined");
-  }
   return <route.Component {...componentProps} />;
-};
-
-AppPage.isBookingPage = ({ router }) => {
-  const route = getRoute(router.query.slug as string, router.query.pages as string[]);
-  if (route.notFound) {
-    return false;
-  }
-  const isBookingPage = route.Component.isBookingPage;
-  if (typeof isBookingPage === "function") {
-    return isBookingPage({ router });
-  }
-
-  return !!isBookingPage;
-};
```

Comment:
This can be removed because RoutingLink is the only component with `isBookingPage = true`, and I handled this by adding `/apps/routing-forms/routing-link` to `useIsBookingPageHook`